Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Components/Sidebar Header Avatar.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Sidebar Header Avatar.swift
+//  Mlem
+//
+//  Created by Jake Shirley on 6/21/23.
+//
+
+import Foundation
+
+
+import SwiftUI
+import CachedAsyncImage
+
+struct CommunitySidebarHeaderAvatar : View {
+    @State var imageUrl: URL?
+    
+    var body: some View {
+        ZStack {
+            if let avatarURL = imageUrl {
+                CachedAsyncImage(url: avatarURL) { image in
+                    image
+                        .resizable()
+                        .clipShape(Circle())
+                        .overlay(Circle()
+                        .stroke(.secondary, lineWidth: 2))
+                } placeholder: {
+                    ProgressView()
+                }
+            } else {
+                // TODO: Default avatar?
```

Comment:
@BrooklynMan - do you know if we have a default avatar we'd like to use here when the user has not set one?